jaeger_client/config.py

```diff
@@ -296,6 +296,16 @@ def initialize_tracer(self, io_loop=None):
                 return
             Config._initialized = True
 
+        tracer = self.new_tracer(io_loop)
+
+        self._initialize_global_tracer(tracer=tracer)
+        return tracer
+
+    def new_tracer(self, io_loop=None):
+        """
+        Create a new Jaeger Tracer based on the passed `jaeger_client.Config`.
+        Does not set `opentracing.tracer` global variable.
+        """
         channel = self._create_local_agent_channel(io_loop=io_loop)
```

Comment:
Only downside here is that we create a separate channel for each tracer, when they could(?) use the same one. However, since the client can pass in an io loop it's not a trivial fix. Not sure how much of an issue this is.